ini/stm32-common.ini

```diff
@@ -0,0 +1,41 @@
+#
+# Marlin Firmware
+# PlatformIO Configuration File
+#
+
+####################################
+#
+# HAL/STM32 Common Environments
+#
+####################################
+
+[common_stm32]
+platform         = ststm32@~12.1
+board_build.core = stm32
+build_flags      = ${common.build_flags}
+                   -std=gnu++14
+                   -DUSBCON -DUSBD_USE_CDC
```

Comment:
unsure USBCON is common ... that means usb handled by the stm32 chip directly... From my point of view, its still kinda unstable to do that...

Reply:
That would have to be addressed in a new PR. This PR should not be making this kind of change, but should be aiming to disrupt the current build options as little as possible.